### PR TITLE
fix(query): add a missing SSE way for SQS

### PR DESCRIPTION
**Pull request by jycamier**
## Context

From [the documentation](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/sqs_queue#server-side-encryption-sse), there are 2 way to add SSE on a AWS SQS : 
1)  SSE-SQS
2)  SSE-KMS

Currently, the query `terraform/aws/sqs_with_sse_disabled` check only SSE-KMS.

## Proposed Changes
- Update the query `terraform/aws/sqs_with_sse_disabled` to check if one of the 2 ways is set and not null

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

File: assets/queries/terraform/aws/sqs_with_sse_disabled/query.rego [hunk 1]
```diff
@@ -2,16 +2,42 @@ package Cx
...
 CxPolicy[result] {
 	resource := input.document[i].resource.aws_sqs_queue[name]
 	not common_lib.valid_key(resource, "kms_master_key_id")
 
+	resource.sqs_managed_sse_enabled == false
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("aws_sqs_queue[%s]", [name]),
+		"issueType": "MissingAttribute",
```
Comment (rafaela-soares) [inline assets/queries/terraform/aws/sqs_with_sse_disabled/query.rego:24]:
```suggestion
		"searchKey": sprintf("aws_sqs_queue[%s].sqs_managed_sse_enabled", [name]),
		"issueType": "IncorrectValue",
```

**Review by rafaela-soares — CHANGES_REQUESTED** [review]
Hello, @jycamier!

Such a great observation and contribution! Thank you so much 🚀 

Please, take a look at the minor suggestions:

**Comment by jycamier** [comment]
Hello @rafaela-soares 
Your welcome ! You're very active and you answer very quickly : it's very pleasant too. 
I like to contribute on Kics and I use it massively.

I've updated / accepted your suggestion change.

**Review by rafaela-soares — APPROVED** [review]
LGTM 🚀 

Thank you so much for the feedback, @jycamier!

We are very grateful for your amazing contributions, suggestions and iterations! 😊 

Let's continue to **K**eep **I**nfrastructure as **C**ode (more) **S**ecure together 🚀

**Pull request merged**